src/Props/faq/faq.ts

```diff
@@ -0,0 +1,69 @@
+import { FAQProps } from "../props"
+
+const FAQInformation: FAQProps[] = [
+  {
+    question: "What is a hackathon?",
+    answer:
+      "UCSC’s hackathon is a three day, weekend-long event where you build your ideas to life, network, and show off your talent. Ideas can be formed through teams or individually. There are also hundreds of students, mentors, sponsors, and judges that can help push your vision forward. Along this event are workshops geared for students of all levels to learn new skills and network with other peers or recruiters.",
+  },
+  {
+    question: "How long is the event?",
+    answer:
+      "CruzHacks is a 3 day weekend event from Friday night to Sunday noon with an options; awards ceremony in the afternoon.",
+  },
+  {
+    question: "Who can participate?",
+    answer:
+      "Anyone and everyone! It is open to anyone who wants to create their vision. No coding or technical experience is necessary. If you are a designer, product manager, artists, business major, etc you are welcome to apply! Mentors will be available to help you through any technical difficulties.",
+  },
+  {
+    question:
+      "Do I need to have a team in place/at all? What would be the limit if so?",
+    answer:
+      "It is not necessary to have a team at the time of application nor at the actual event. Teams must be a max of 4 people.",
+  },
+  {
+    question: "Do I need to know how to code?",
+    answer:
+      "Nope! No experience is required. We’re looking for applicants who are passionate about learning something new and creating a vision. 25% of hackers in 2021 and 47% in 2020 were first time hackers.",
+  },
+  {
+    question: "Do I need to be a Computer Science/Engineering major?",
+    answer: "We welcome and accept people of all majors!",
+  },
+  {
+    question: "Will CruzHacks be virtual or in-person?",
+    answer:
+      "Due to UCSC, Santa Cruz, and California protocols, CruzHacks will remain virtual for CruzHacks 2022. However, there may be a possibility of having a short in-person event for those in the Santa Cruz county that abide by UCSC protocols and feel comfortable to attend.",
+  },
+  {
+    question: "Does it cost anything to participate",
+    answer: "Cruzhacks is free for all participants!",
+  },
```

Comment:
There is a missing question mark here. We are also missing a question in the middle column